common/pipes/pubspec.yaml

```diff
@@ -1,10 +1,11 @@
 name: pipes
-description: Angular Dart API Examples for Pipes
+description: AngularDart API Examples for Pipes
 version: 0.0.1
 environment:
   sdk: '>=1.19.0 <2.0.0'
 dependencies:
-  angular2: 2.0.0-beta.22
```

Comment:
As pointed out in the [pub documentation for the caret syntax](https://www.dartlang.org/tools/pub/dependencies#caret-syntax), `^2.2.0` means
> the range of all versions **guaranteed to be backwards compatible** with [2.2.0]

Why is this not good enough?